CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.13)
+project(fzf C)
+
+set(CMAKE_INSTALL_PREFIX ".")
+
+if (MSVC)
+    set(CMAKE_SHARED_LIBRARY_PREFIX lib)
+    add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+    add_compile_options(/W4)
```

Comment:
I would suggest to call `add_library` first and then use `target_compile_definitions` and `target_compile_options`.